apis/r/tests/testthat/test-SeuratIngest.R

```diff
@@ -26,6 +26,10 @@ test_that("Write Assay mechanics", {
       info = layers[i]
     )
   }
+
+  ms$close()
+  gc()
```

Comment:
Whether `gc()` helps or not is never clear, but your chances will be marginally better if you first `rm()` the objects holding the memory.  (With sufficient logging you can see finalizers entered upon `gc()` _if_ objects have been removed.)  The trouble with `gc()` is that doesn't not help much on fragmented memory.